roles/nextcloud-server/defaults/main.yml

```diff
@@ -75,10 +78,10 @@ nextcloud_nextcloud_apache_container_http_host_bind_port: "{{ '' if nextcloud_ng
 # on this port by default.
 nextcloud_apache_http_port_on_host: 37150
 
-nextcloud_postgres_connection_hostname: "nextcloud-postgres"
 nextcloud_postgres_connection_username: "nextcloud"
 nextcloud_postgres_connection_password: "nextcloud-password"
 nextcloud_postgres_db_name: "nextcloud"
+nextcloud_postgres_connection_hostname: "nextcloud-postgres"
```

Comment:
I updated the order of the variables here to reflect the order required for initializing a new nextcloud instance through the GUI once successfully setup and deployed.